build/deb/ethereum/deb.control

```diff
@@ -11,8 +11,8 @@ Vcs-Browser: https://github.com/ethereum/go-ethereum
 Package: {{.Name}}
 Architecture: any
 Depends: ${misc:Depends}, {{.ExeList}}
-Description: Meta-package to install geth, swarm, and other tools
- Meta-package to install geth, swarm and other tools
+Description: Meta-package to install geth, swarm and other tools
```

Comment:
Please remove `, swarm`

Reply:
I think `ethereum-swarm` is still part of the `ethereum` metapackage, no?

Not sure if we should remove it, but if we are keeping the current behaviour (i.e. installing geth, swarm, etc. if you install `ethereum`), then this should stay I think.

Reply:
On a second though, it seems that we have removed `ethereum-swarm` from the meta package, so actually @karalabe is correct :) `ethereum-swarm` is no longer installed as part of `ethereum`, so we should remove all references of `swarm`.